components/automate-deployment/pkg/manifest/provider.go

```diff
@@ -2,21 +2,31 @@ package manifest
 
 import (
 	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"sort"
+	"strconv"
+	"strings"
 	"time"
 
+	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 
 	"github.com/chef/automate/components/automate-deployment/pkg/habpkg"
 	"github.com/chef/automate/components/automate-deployment/pkg/services"
 )
 
+const (
+	automateVersionsURLFmt = "https://packages.chef.io/manifests/%s/automate/versions.json"
```

Comment:
should this live in http.go along with the other manifest urls,  instead of here?

Reply:
No @rickmarry, that should go to http.go only when we are implementing this at the client level. But here we are implementing at the manifest provider level, and not at the client level, so this should be here only.